{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"internalServiceErrorException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/InternalServiceErrorException","preciseIdentifier":"s:12AWSAppRunner29InternalServiceErrorExceptionV","text":"InternalServiceErrorException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsapprunner\/listvpcingressconnectionsoutputerror\/internalserviceerrorexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/ListVpcIngressConnectionsOutputError\/internalServiceErrorException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"internalServiceErrorException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"InternalServiceErrorException","preciseIdentifier":"s:12AWSAppRunner29InternalServiceErrorExceptionV"},{"kind":"text","text":")"}],"title":"ListVpcIngressConnectionsOutputError.internalServiceErrorException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:12AWSAppRunner36ListVpcIngressConnectionsOutputErrorO015internalServiceH9ExceptionyAcA08InternaljhK0VcACmF","modules":[{"name":"AWSAppRunner"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner","doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/ListVpcIngressConnectionsOutputError"]]},"references":{"doc://AWSAppRunner/documentation/AWSAppRunner/ListVpcIngressConnectionsOutputError":{"role":"symbol","title":"ListVpcIngressConnectionsOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ListVpcIngressConnectionsOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/ListVpcIngressConnectionsOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ListVpcIngressConnectionsOutputError"}],"url":"\/documentation\/awsapprunner\/listvpcingressconnectionsoutputerror"},"doc://AWSAppRunner/documentation/AWSAppRunner":{"role":"collection","title":"AWSAppRunner","abstract":[],"identifier":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner","kind":"symbol","type":"topic","url":"\/documentation\/awsapprunner"},"doc://AWSAppRunner/documentation/AWSAppRunner/InternalServiceErrorException":{"role":"symbol","title":"InternalServiceErrorException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalServiceErrorException"}],"abstract":[{"type":"text","text":"An unexpected service exception occurred."}],"identifier":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/InternalServiceErrorException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"InternalServiceErrorException"}],"url":"\/documentation\/awsapprunner\/internalserviceerrorexception"},"doc://AWSAppRunner/documentation/AWSAppRunner/ListVpcIngressConnectionsOutputError/internalServiceErrorException(_:)":{"role":"symbol","title":"ListVpcIngressConnectionsOutputError.internalServiceErrorException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"internalServiceErrorException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"InternalServiceErrorException","preciseIdentifier":"s:12AWSAppRunner29InternalServiceErrorExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAppRunner\/documentation\/AWSAppRunner\/ListVpcIngressConnectionsOutputError\/internalServiceErrorException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsapprunner\/listvpcingressconnectionsoutputerror\/internalserviceerrorexception(_:)"}}}